test/lib/ansible_test/_data/collection_metadata.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""Retrieve some collection metadata."""
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+import os
+import sys
+
+try:
+    import yaml
+except ImportError:
+    yaml = None
+
+
+def load_metadata(collection_path):
+    """
+    Retrieve some collection metadata.
+    """
+    result = dict(warnings=[])
+
+    # Try reading galaxy.yml
```

Comment:
If both exist, should the MANIFEST.json take precedence over a galaxy.yml file instead? The manifest indicates a built/installed collection artifact so `ansible-galaxy` is going to prefer the artifact data and fall back to galaxy.yml.

Reply:
Good idea. That's in the code from #66920 though, so I'll fix it there first and then rebase.

Reply:
In this PR, it's now in 71c4471b44c4e0bdf65ea2bdf1f5bf6e2329979c.